src/odemis/gui/model/__init__.py

```diff
@@ -954,21 +955,22 @@ def __init__(self, a_file=None, metadata=None):
             self.file_name = a_file.name
             self.file_obj = a_file # file object
 
-        # Ensure the file name contains the full path
-        self.file_name = os.path.abspath(self.file_name)
-
         # TODO: settings of the instruments for the acquisition?
         # Might be per stream
         self.metadata = metadata or {}
 
-        if model.MD_ACQ_DATE not in self.metadata and self.file_name:
-            # try to auto fill acquisition time (seconds from epoch)
-            try:
-                acq_date = os.stat(self.file_name).st_ctime
-                self.metadata[model.MD_ACQ_DATE] = acq_date
-            except OSError:
-                # can't open the file => just cannot guess the time
-                pass
+        if not any(self.file_name.startswith(p) for p in catmaid.PREFIXES):
```

Comment:
Could you try to make it work without using catmaid? Actually I have a feeling that the original code would work fine. What's wrong with setting "file_name"? If other things break due to filename being "weird", I'd suggest to generically either make them work, or modify the filename on the file to make it less "weird".

Reply:
I removed this, it doesn't depend on Catmaid anymore. I thought the filename broke things, but it didn't.